geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/Coder.java

```diff
@@ -83,6 +83,9 @@
   @MakeImmutable
   public static final byte[] oom = stringToBytes("OOM ");
 
+  @MakeImmutable
+  public static final byte[] BUSYKEY = stringToBytes("BUSYKEY ");
+
```

Comment:
Could this be moved to the `StringBytesGlossary` class after rebasing on develop?

Reply:
Done